addons/acodec/acodec.c

```diff
@@ -38,6 +38,48 @@ bool al_init_acodec_addon(void)
 #endif
 
 #ifdef ALLEGRO_CFG_ACODEC_MODAUDIO
+#include <dumb.h>
```

Comment:
Probably shouldn't include a header inside the body of a function.

Reply:
Good catch, thanks.